package.json

```diff
@@ -15,8 +15,8 @@
     "dev": "turbo run dev --parallel",
     "clean": "turbo run clean --parallel",
     "format": "turbo run format && node scripts/format.mjs",
-    "lint": "turbo run lint",
-    "lintfix": "turbo run lintfix",
+    "lint": "turbo run lint; eslint ./cypress/**/*.ts",
```

Comment:
Should we instead setup a `package.json` inside the `cypress` folder, move all cypress related `scripts` and `devDependencies` in there, and add the folder to `pnpm-workspace.yaml`?